suite-common/wallet-core/src/accounts/accountsReducer.ts

```diff
@@ -86,15 +87,35 @@ export const prepareAccountsReducer = createReducerWithExtraDeps(
 
 export const selectAccounts = (state: AccountsRootState) => state.wallet.accounts;
 
-export const selectAccount = (state: AccountsRootState, accountKey: string) =>
-    state.wallet.accounts.find(account => account.key === accountKey);
+export const selectAccount = createSelector(
```

Comment:
According to this proposal - https://github.com/trezor/trezor-suite/pull/6268#discussion_r972130355 - I am voting for:

`selectAccount => selectAccountByKey`

Reply:
It looks good, only thing that is little bit confusing that we somewhere call it `accountKey` and somewhere `accountHash` can we unify that?

Reply:
Yes. As I can see, `accountKey` is a standardized name in suite. So I replaced all accountHash occurrences in e6ad9e8b2732218f1a2ddefcff55baba167db860.